tests/test_create_actions_config.py

```diff
@@ -0,0 +1,38 @@
+"""This module tests the githubactions module"""
+import os
+
+from configator import create_actions_config
+
+
+def test_create_configator_file_creates_github_actions(mocker):
+    """ Testing to see if a file is created through github_actions"""
+    mock_open = mocker.mock_open()
+    #take in buildin open function and replace with mock function
+    mocker.patch('builtins.open', mock_open)
+    mocker.patch("os.makedirs")
+    create_actions_config.create_configator_file()
+    mock_open.assert_called_once_with(".github/workflows/grade.yml","w", encoding='utf-8')
+
+def test_create_configator_file_creates_folder_structure(mocker):
+    """ Testing to see if a folder is created through makedirs"""
+    mock_open = mocker.mock_open()
+    #take in buildin open function and replace with mock function
+    mocker.patch('builtins.open', mock_open)
+    mocker.patch("os.makedirs")
+    create_actions_config.create_configator_file()
+    os.makedirs.assert_called_once_with(".github/workflows") # pylint: disable=E1101
+
+#def test_create_create_configator_file_has_text_inside(mocker):
```

Comment:
You should expand this commented-out test case to check the contents of `grade.yml` in some way -- you could use `tests/test_configator.py:test_create_gradlebuild_writes_through_gradlebuild` as an example for testing written file output.

Reply:
Specifically, you should check if `GatorEducator/gatorgradle-action` will be written to the file.